llarp/context.cpp

```diff
@@ -433,16 +406,16 @@ extern "C"
   struct llarp_main *
   llarp_main_init(const char *fname, bool multiProcess)
   {
+    (void)multiProcess;
```

Comment:
don't we have abseil unused something or another?

Reply:
alternatively, should this argument go away because it's no longer going to be relevant?

Reply:
Ideally argument goes away imo.